Bot/Discord.Bot.Extensions/Cea/Commands/CeaSlashCommand.cs

```diff
@@ -0,0 +1,89 @@
+﻿using Discord.Bot;
+using Discord.WebSocket;
+using PlayCEAStats.DataModel;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Discord.Cea
+{
+    public class CeaSlashCommand : SlashCommand
+    {
+        public override string Name => "cea";
+
+        public override string Description => "Command to see info from PlayCea.com.";
+
+        public override bool DefaultPermissions => true;
+
+        public override bool IsGlobal => true;
+
+        public override Dictionary<ulong, List<ApplicationCommandPermission>> GuildIdsAndPermissions => new Dictionary<ulong, List<ApplicationCommandPermission>>()
+        {
+        };
+
+        private Dictionary<string, ICeaSubCommand> subCommands;
+
+        public CeaSlashCommand() : base()
+        {
+            List<ICeaSubCommand> subCommands = new()
+            {
+                new CeaTeamCommand()
+            };
+
+            this.subCommands = subCommands.ToDictionary(c => c.OptionBuilder.Name);
+        }
+
+        public override async Task HandleCommandAsync(SocketSlashCommand command, DiscordSocketClient client)
+        {
+            var subCommand = command.Data.Options.Where(o => o.Type.Equals(ApplicationCommandOptionType.SubCommand)).First();
```

Comment:
I like how you are handling sub commands. We should follow something cleaner like this in other commands.